src/main/java/models/Game.java

```diff
@@ -7,25 +7,27 @@
 /**
  * Created by michaelhilton on 1/25/16.
  */
-public class Game {
+public abstract class Game {
 
     public java.util.List<Card> deck = new ArrayList<>();
 
     public java.util.List<java.util.List<Card>> cols = new ArrayList<>();
 
-    public int score;
+    public int score = 0;
 
-    public int cardsLeft;
+    public int cardsLeft = 0;
 
     public Game(){
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
+        this.buildDeck();
```

Comment:
Thank you for your reply. And sorry for making confusing you. My English must be terrible.

I am sorry if I am wrong, but you may not notice that my comment is line note (comment in "Files changed" page).
I was just curious that you didn't put shuffle() in the constructor, while you put buildDeck() in it.
I think there will be no difference if put them(shuffle and buildDeck) in constructor or call them in application controller, so I wondered why.